### ISSUE TITLE
508-defect-2 [SCREENREADER]: The map is announcing a lot of extra data on focus and each droplet needs a usable name

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#515]
## Issue
* As a screen reader user, I want to hear relevant information when the map receives keyboard focus.The droplets/pins all read out "Try using our other tools to search, link" when they receive keyboard focus. These were flagged as SC 4.1.2 violations. \

## Audit Finding
* Note 9, Defect 5 and 6

## Acceptance Criteria
- [ ] When the map receives keyboard focus, each pin should be treated as a paragraph, presenting information from the result card (aria-described by)
- [ ]  When a pin is selected, the corresponding search result card receives focus and the screenreader presents information from the result card (aria-described by) 
- [ ]  When a pin is selected on the map in mobile, information about the facility is displayed

**Please consider using a feature flag for this complicated work as it will need QA**

## Environment
* MacOS Mojave
* Chrome latest
* https://staging.va.gov/find-locations/

<details>
 <summary> Possible Fixes (optional) </summary>
There will be two fixes for this item. The two snippets below show what the map and the icon could read out when they take keyboard focus.

```html
<!-- Containing div -->
<div
  class="leaflet-container leaflet-touch leaflet-retina leaflet-fade-anim leaflet-grab leaflet-touch-drag leaflet-touch-zoom"
  style="min-height: 75vh; width: 100%; position: relative;"
  tabindex="0"
>
  <span class="sr-only">You are viewing 5 results within 10 miles of Austin, TX<span>
  ...
</div>
```

```html
<!-- Map icons -->
<div
  class="leaflet-marker-icon leaflet-zoom-animated leaflet-interactive"
  style="margin-left: -6px; margin-top: -6px; width: 12px; height: 12px; transform: translate3d(343px, 345px, 0px); z-index: 345;"
  tabindex="0"
>
  <div class="vet-center-icon map-marker">
    <span class="sr-only">Austin VA Clinic</span>
  </div>
</div>
```

## WCAG or Vendor Guidance (optional)
* [Name, Role, Value: Understanding SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)
</details>

**Comment by mmiddaugh** [comment]
May be more than one issue
Needs discovery and/or conversation with Trevor
@brangi

**Comment by mmiddaugh** [comment]
Michelle will schedule time with Jennifer to walk through this issue and understand ideal user experience so we can determine tasks to resolve.

**Comment by mmiddaugh** [comment]
In #14872, we re-evaluated the new GL JS implementation against existing accessibility defects and desired behavior. It was determined that the plug-in is not more than a javascript class that modifies styles that do not apply to our implementation.

**Comment by mmiddaugh** [comment]
This should be resolved by 508-defect-2 Mapbox in keyboard user flow #16971

**Comment by mmiddaugh** [comment]
Closing this as resolved by #16971
